gcp_deepvariant_runner.py

```diff
@@ -57,7 +57,7 @@
 import os
 import re
 import subprocess
-import urlparse
+from urllib.parse import urlparse
```

Comment:
import `parse` from `urllib` and call `urlparse` from it.

Reply:
That change will cause a lint error:
go/gpylint-faq#g-importing-member

https://g3doc.corp.google.com/eng/doc/devguide/py/style/index.md?cl=head#imports

Reply:
huh, so using 

```
from urllib import parse

parse.urlparse(...)
```
causes pylint error?! that is weird, I would imagine that `parse` _is_ the module we are supposed to use...

I'm curious then, in order to accommodate our guideline, whether we then need to write it as 
```
import urllib

urllib.parse.urlparse(...)
```

Reply:
Done.